### PR TITLE
add a Services page, and a Contact page

### PR DESCRIPTION
**Pull request by sethladd**
Not ready for commit. Just crowd-sourcing links.

### CODE REVIEW

File: services.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+The following services are available for Flutter, but
+require an additional installation step.
+
+### Cross platform
+
+* Firebase
+  * [docs](https://github.com/flutter/firebase-dart/blob/master/README.md)
+  * [example](https://github.com/flutter/firechat-flutter)
+* Google Cloud Messaging (GCM)
+  * docs (coming soon)
```
Comment (mpcomplete) [inline services.md]: Just fixed up some docs: https://github.com/flutter/flutter_gcm/blob/master/README.md

File: services.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The following services are available for Flutter, but
+require an additional installation step.
+
+### Cross platform
+
+* Firebase
+  * [docs](https://github.com/flutter/firebase-dart/blob/master/README.md)
+  * [example](https://github.com/flutter/firechat-flutter)
+* Google Cloud Messaging (GCM)
```
Comment (mpcomplete) [inline services.md]: BTW, ios support for GCM is not implemented. Maybe put this under android-only for now.

**Comment by Hixie** [comment]
@sethladd Were there specific links you were looking for?

**Comment by sethladd** [comment]
Yes. A link to the docs and ideally an example/snippet/sample for each service. Goal is to have a canonical list of "things I can do with flutter, if I want to do more than show material design widgets" :)

**Comment by Hixie** [comment]
@sethladd Since nothing is happening on this PR, I'm going to close it so that it doesn't appear on our list of PRs pending review. Let me know if that's the wrong thing to do. Thanks.

**Comment by sethladd** [comment]
No worries. I'll reopen when I have more links.

**Pull request closed without merging**